gems/pending/appliance_console/database_replication_primary.rb

```diff
@@ -0,0 +1,45 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationPrimary < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr master register'.freeze
+
+    def initialize
+      set_defaults
+    end
+
+    def set_defaults
```

Comment:
Why have a separate method for this if `initialize` is the only place it is called?